### ISSUE TITLE
Sample battery voltage at interval

### ISSUE THREAD

**Issue by ojousima** [ruuvi/ruuvi.firmware.c#87]
Current battery voltage sampling gets triggered on every radio transmission, which is unnecessarily often. 
A logic to update the voltage only once per minute, with interval configurable in `application_mode_default.h` should be added.
Related to #86 
```
/** @brief Resample battery voltage at this interval */
#ifndef APP_BATTERY_SAMPLE_MS
#   define APP_BATTERY__SAMPLEL_MS (60U*1000U)
#endif
```

**Comment by ojousima** [comment]
Fixed in #92, marking as pending. Thanks

**Comment by ojousima** [comment]
Done in 7a9da7c
